lib/datadog/core/configuration/settings.rb

```diff
@@ -497,6 +497,17 @@ def initialize(*_)
             o.default { env_to_float(Core::Remote::Ext::ENV_POLL_INTERVAL_SECONDS, 5.0) }
             o.lazy
           end
+
+          # Declare service name to bind to remote configuration. Use when
+          # DD_SERVICE does not match the correct integration for which remote
+          # configuration applies.
+          #
+          # @default `nil`.
+          # @return [String,nil]
+          option :service do |o|
+            o.default { nil }
+            o.lazy
+          end
```

Comment:
Do we need to set `default` and `lazy`? It seems we can simply have `option :service`

Reply:
Indeed, fixed